backbone.js

```diff
@@ -808,7 +808,10 @@
       var singular = !_.isArray(models);
       models = singular ? [models] : _.clone(models);
       var removed = this._removeModels(models, options);
-      if (!options.silent && removed) this.trigger('update', this, options);
+      if (!options.silent && removed) {
+        options.removedModels = removed;
```

Comment:
Why is this one `options.removedModels` but below it's `options.changes.removed`?